src/constants.rs

```diff
@@ -160,6 +160,8 @@ pub const DC_MSG_ID_LAST_SPECIAL: u32 = 9;
 pub const DC_MAX_GET_TEXT_LEN: usize = 30000;
 /// approx. max. length returned by dc_get_msg_info()
 pub const DC_MAX_GET_INFO_LEN: usize = 100_000;
+/// approx. max. length returned by dc_msg_get_error()
+pub const DC_MAX_GET_ERROR_LEN: usize = 10000;
```

Comment:
Why do you think you need to truncate the error? These limitations look like they come from the C core.

Reply:
you are absolutely right that this isn't neccessary!